src/guild/config/guildConfigs.ts

```diff
@@ -18,9 +18,9 @@ guildConfigs.set('239599059415859200', {
 		superLeague: '470655938018279448',
 		youTube: '588769186621161503',
 		buildTeam: '502570149157797898',
-		support: '726663955065470997',
+		helper: '726663955065470997',
 		robot: '394262717663412224',
-		juniorModerator: '276887063683399680',
+		superhelper: '276887063683399680',
```

Comment:
Would make `superhelper` camel case to be consistent so it would be `superHelper`